README.md

```diff
@@ -2676,10 +2676,23 @@ A successful response uses the HTTP code **200 OK** and has a JSON body with the
 | Field                   | Value                           | Description      |
 |:------------------------|:--------------------------------|:-----------------|
 | `result`                | String                          | The value `success` indicates that the body represents a successful response. |
-| `last_datetime`         | Integer                         | The last reported validation vote signed by this validator. |
 | `validation_public_key` | String - Base-58 [Public Key][] | This validator's validator public key. |
 | `domain`                | String                          | (May be omitted) The DNS domain associated with this validator. |
-| `domain_state`          | String                          | The value `verified` indicates that this validator has a [verified domain](https://developers.ripple.com/run-rippled-as-a-validator.html#domain-verification) controlled by the same operator as the validator. The value `AccountDomainNotFound` indicates that the "Account Domain" part of Domain Verification is not set up properly. The value `RippleTxtNotFound` indicates that the ripple.txt step of Domain Verification is not set up properly. |
+| `chain`                 | String                          | Ledger hash chain which this validator is currently following. |
+| `unl`                   | Bool                            | True if the validator is part of the ledger chain's UNL. |
```

Comment:
`s/UNL/recommended UNL/`

Reply:
I don't understand this comment

Reply:
I suggest you substitute "UNL" for "recommended UNL"